contents/projectManagement/AgileDevelopment.md

```diff
@@ -0,0 +1,127 @@
+
+## What is agile development?
+
+Popularised in 2001 by the [*Manifesto for Agile Software Development*](http://agilemanifesto.org/), agile development is essentially a software development method that is flexible, iterative and people-centric.
+
+But what does "flexible, iterative and people-centric" really mean? To comprehend this, let us examine each characteristic individually.
+
+---
+
+For a start, agile development is "flexible" because it gives a project more room to evolve over time. This is possible as agile development utilises a form of planning known as [adaptive planning](https://www.projectmanager.com/blog/how-to-plan-in-an-agile-environment), which means that planning is done continuously throughout a project rather than sticking rigidly to a fixed plan established at the start. As a result, planning can take into consideration the project's current progress, and change accordingly to allow the project's priorities and scope to evolve.
+
+To visualise this, let us consider an everyday scenario below:
+
+>Suppose a planner intends to transfer between airplanes, and the planner’s normal plan is to:
+>1. Exit first airplane via arrival gate.
+>2. Determine departure gate.
+>3. Walk to the departure gate.
+>4. Board second airplane via departure gate.
+>In a small airport this prestored plan would work just fine. But in a larger airport, such as >Kennedy Airport, if the arrival and departure gates are in different terminals, the plan would >have to be refitted (e.g., the planner might have to take a shuttle between terminals). The >problem of adaptive planning (reasoning) is to take a prestored plan (piece of knowledge) such
+>as the one shown above, and apply it to a novel set of circumstances...
+>
+>-- from [Adaptive Planning](http://onlinelibrary.wiley.com/store/10.1207/s15516709cog1203_3/asset/s15516709cog1203_3.pdf?v=1&t=jeef0qzp&s=ce2d7fe621eee0a3d7a4a25867e84fb22668d304) by Richard Alterman
+
+---
+
+Secondly, agile development is "iterative" because it breaks product development work down into short cycles known as [iterations](https://www.agilealliance.org/glossary/iteration/). Each iteration involves going through planning, designing, coding and testing to result in a working product at the end of an iteration. However, a single iteration may not have added enough functionality to warrant a new release, so in reality multiple iterations are often required to release a new product or feature. Here is a diagram to illustrate this concept:
+
+![Agile Development - Iterations](images/projectManagement-agileDev-iterations.jpg)
+
+---
+
+Thirdly, agile development is "people-centric" because it focuses heavily on collaboration and effective communication. In fact, there are many recommendations on how to form a good agile team, but the general consensus is that agile development involves [several crucial team roles](https://apiumhub.com/tech-blog-barcelona/building-agile-team/) that work in close relation with each other.
```

Comment:
Since agile has different frameworks under it, the "team roles" does it apply broadly to all the frameworks? The link seems to suggest it is for scrum.